GliaWidgets/Sources/View/Chat/Message/SystemMessageView.swift

```diff
@@ -28,10 +28,12 @@ final class SystemMessageView: ChatMessageView {
     private func layout() {
         addSubview(contentViews)
 
-        var constraints = [NSLayoutConstraint](); defer { constraints.activate() }
-        constraints += contentViews.layoutInSuperview(edges: .vertical, insets: kInsets)
-        constraints += contentViews.layoutInSuperview(edges: .top, insets: kInsets)
-        constraints += contentViews.trailingAnchor.constraint(lessThanOrEqualTo: trailingAnchor, constant: kInsets.right)
+        NSLayoutConstraint.activate([
+            contentViews.leadingAnchor.constraint(equalTo: self.leadingAnchor, constant: kInsets.left),
+            contentViews.trailingAnchor.constraint(lessThanOrEqualTo: self.trailingAnchor, constant: -kInsets.right),
+            contentViews.topAnchor.constraint(equalTo: self.topAnchor, constant: kInsets.top),
+            contentViews.bottomAnchor.constraint(equalTo: self.bottomAnchor, constant: -kInsets.bottom).priority(.required)
```

Comment:
Is the `required` priority default?

Reply:
Yes, I think this was in initial pre-PureLayout removal implementation. Removed e47dd9cb6ae1770591bd789b810b4812271e1e04.